test/IISIntegration.FunctionalTests/IISIntegration.FunctionalTests.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
```

Comment:
It would stop building other targets of functional test sites, is it a problem?

Reply:
Only for inner loop dev. Removing the extra TFM avoids running the tests multiple times.